src/ct/ct_menu_actions.cc

```diff
@@ -293,6 +293,8 @@ void CtMenu::init_actions(CtActions* pActions)
         _("Replace Tabs with Spaces"), sigc::mem_fun(*pActions, &CtActions::replace_tabs_with_spaces)});
     _actions.push_back(CtMenuAction{tools_cat, "command_palette", "ct_execute", _("_Command Palette..."), KB_CONTROL+KB_SHIFT+"p",
         _("Command Palette"), sigc::mem_fun(*pActions, &CtActions::command_palette)});
+    _actions.push_back(CtMenuAction{tools_cat, "select_node", "ct_execute", _("_Select Node..."), KB_CONTROL+KB_ALT+"p",
+        _("Select Node"), sigc::mem_fun(*pActions, &CtActions::command_selnode)});
```

Comment:
there is an issue with the windows gtk library so that when the user presses AltGr it detects Ctrl+Alt (causing unwanted clashes on non english keyboard layouts characters input), because of this we cannot use any Ctrl+Alt keyboard shortcuts. You can leave it without a shortcut for now so that the user individually can assign one, I will look for another free one